### PR TITLE
nit: use .rst links in readme & typo

### PR DESCRIPTION
**Pull request by itstemo**
# TL;DR

looks like the readme is in RST, but using markdown links `[bla](foo.com)`. Switched to use .rst links `bla <foo.com>_`, and fixed a couple typos.

## Type
 - [ ] Bug Fix
 - [ ] Feature
 - [ ] Plugin
Docs change 
## Are all requirements met?
N/A
 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
N/A
https://github.com/lyft/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/lyft/flyte/issues/<number>_

### CODE REVIEW

**Comment by itstemo** [comment]
cc 👀 @EngHabu , couldn't request a pr review directly -- just a small readme change. tagging since it looks like you recently the readme as well. 

feel free to close if not needed

**Comment by kumare3** [comment]
@itstemo so sorry, we somehow missed getting notified about this message. Also for some reason the README.md seems to have vanished. Lets convert this to README.md instead of rst - for consistency across all repos

**Comment by kumare3** [comment]
Also you need to sign DCO

**Review by EngHabu — APPROVED** [review]

**Comment by kumare3** [comment]
I have created this PR - https://github.com/flyteorg/flyteidl/pull/192

**Comment by EngHabu** [comment]
@itstemo Thank you for your contribution! and helping Flyte be more friendly to contributors... deleting the Readme.md was unintended as part of a larger cleanup to streamline the generation for docs. Ketan is bringing that back in #192.
